rome-modules/src/main/java/com/rometools/modules/mediarss/io/MediaModuleParser.java

```diff
@@ -92,6 +81,30 @@ public Module parse(final Element mmRoot, final Locale locale) {
         return mod;
     }
 
+
+    private long parseFileSize(String fileSize) {
+        Matcher matcher = Pattern.compile("([\\d,.]+)([TGMK])?B", Pattern.CASE_INSENSITIVE).matcher(fileSize);
+        if (matcher.matches()) {
+            BigDecimal number = new BigDecimal(matcher.group(1).replace(',', '.'));
+            BigDecimal multiplier = BigDecimal.valueOf(1);
+            if (matcher.group(2) != null) {
+                char unit  = matcher.group(2).toLowerCase().charAt(0);
+                if (unit == 'k') {
+                    multiplier = BigDecimal.valueOf(1024);
```

Comment:
So it seems. Thought i did some proper sampling on this earlier, but some more proper testing now indicates 1000 being the sane choice even though I've seen both being used in the wild. What SI/IEC defines as a KB and KiB doesn't really seem relevant though, as file sizes historically has used 1024 as KB while transfer speeds as been multiples of 1000. This seems like an edge case :)